docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -24,9 +24,7 @@ sequenceDiagram
     activate Solver
     Solver-->>Autopilot: Score
     deactivate Solver
-    Autopilot->>Solver: /reveal
     activate Solver
-    Solver-->>Autopilot: Solution with call data
     deactivate Solver
```

Comment:
This can be removed, right?
Looks like this results in a bug in the generated diagramm.